plugin-tester-scala/src/main/protobuf/helloworld.proto

```diff
@@ -1,7 +1,7 @@
 syntax = "proto3";
 
 option java_multiple_files = true;
-option java_package = "io.grpc.examples";
+option java_package = "io.grpc.examples.helloworld";
```

Comment:
This is more consistent with upstream though, so OK... we should get our `helloworld.proto`s consistent again though :)